seed/SeedButton/SeedButton.cpp

```diff
@@ -0,0 +1,36 @@
+#include "daisy_seed.h"
+
+// Use the daisy namespace to prevent having to type 
+// daisy:: before all libdaisy functions
+using namespace daisy;
+
+// Declare a DaisySeed object called hardware
+DaisySeed hw;
+
+int main(void)
+{
+    // Configure and Initialize the Daisy Seed
+    // These are separate to allow reconfiguration of any of the internal
+    // components before initialization.
+    hw.Configure();
+    hw.Init();
+    
+    //Configure and initialize button
+    Switch button1;
+    //Set button to pin 28, to be updated at a 1kHz  samplerate
+    button1.Init(hw.GetPin(28),1000);
+    //Start timer class, used to enforce 1kHz samplerate
+    dsy_tim_start();
```

Comment:
use `dsy_system_delay()` instead of using the tim based delays here. system delay is always based on ms, and requires no init.